src/generate-signed-data.tsx

```diff
@@ -0,0 +1,21 @@
+import { signSmartContractData } from '@wert-io/widget-sc-signer';
+type SignSmartContractDataParameters = Parameters<
+  typeof signSmartContractData
+>[0];
+
+export type SmartContractOptions = SignSmartContractDataParameters & {
```

Comment:
I've decided to make this function expect a single object as an argument to make it simpler for our clients. Do you think it's okay?